src/getConfig.js

```diff
@@ -0,0 +1,140 @@
+/* eslint no-console: 0 */
+/* eslint no-prototype-builtins: 0 */
+const chalk = require('chalk')
+const format = require('stringify-object')
+const intersection = require('lodash/intersection')
+const defaultsDeep = require('lodash/defaultsDeep')
+const isObject = require('lodash/isObject')
+const validate = require('jest-validate').validate
+const unknownOptionWarning = require('jest-validate/build/warnings').unknownOptionWarning
+const logValidationWarning = require('jest-validate/build/utils').logValidationWarning
+const isGlob = require('is-glob')
+
+/**
+ * Default config object
+ *
+ * @type {{concurrent: boolean, chunkSize: number, gitDir: string, globOptions: {matchBase: boolean, dot: boolean}, linters: {}, subTaskConcurrency: number, renderer: string, verbose: boolean}}
+ */
+const defaultConfig = {
+  concurrent: true,
+  chunkSize: Number.MAX_SAFE_INTEGER,
+  gitDir: '.',
+  globOptions: {
+    matchBase: true,
+    dot: true
+  },
+  linters: {},
```

Comment:
Question: should we set linters to `null` or `undefined` initially so that the validation fails on it (making them a required property)?

Reply:
I think it's better to set it to `undefined`

Reply:
It may be a good idea.

Reply:
I would suggest a `lodash#isEmpty` check because that also validates an empty config object.

Reply:
Unfortunately, setting it to `undefined` doesn't make `jest-validate` to fail the validation. I'm not sure there is way to do so using it. This can be improved afterwards I think.